solr/htdocs/solr/template_data.js

```diff
@@ -434,16 +434,21 @@
       },
       postproc: function(el, data) {
         $(document).on('maybe_update_state', function(e, change, incr) {
-          var ref1;
-          $.getJSON("/Multi/Ajax/psychic", {
-            q: (ref1 = change.q) != null ? ref1 : ''
-          }, function(data) {
+          var facet_species, query_params, ref1, ref2, ref3;
+          facet_species = data != null ? (ref1 = data.state) != null ? (ref2 = ref1.hub) != null ? (ref3 = ref2.params) != null ? ref3.facet_species : void 0 : void 0 : void 0 : void 0;
```

Comment:
That is a very nested condition, is it possible to break it down (dont know if you want to use a switch case) but simple if else will do. makes it clearer to read and understand

Reply:
This is an automatically generated file!

Reply:
ok then ignore my comments for this file. if these has been generated then they should be fine. But still i will try removing all .js files and restart the server just to check the minification doesnt fail. Just as a precaution.